buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -65,8 +67,10 @@
     private static final Logger LOGGER = Logging.getLogger(ElasticsearchNode.class);
     private static final int ES_DESTROY_TIMEOUT = 20;
     private static final TimeUnit ES_DESTROY_TIMEOUT_UNIT = TimeUnit.SECONDS;
-    private static final int NODE_UP_TIMEOUT = 60;
-    private static final TimeUnit NODE_UP_TIMEOUT_UNIT = TimeUnit.SECONDS;
+    private static final int NODE_UP_TIMEOUT = 2;
+    private static final TimeUnit NODE_UP_TIMEOUT_UNIT = TimeUnit.MINUTES;
+    private static final int ADDITIONAL_CONFIG_TIMEOUT = 15;
```

Comment:
15 seconds for some of these things seems like a lot, no? Do we expect just adding something to the keystore or creating a user to be that slow?

Reply:
I have seen it take close to that in testing. 
We are starting up a JVM for it which gets quite slow under high load. 
Going higher with these  timeouts is what gives fair confidence that 
pr-2 will be stable, as we can't really control how many clusters are running.
Individual operations will be slower ( cumulative CPU time is ~ 6 hours ) but the build still completes overall faster because we are using all CPUs.